src/components/connexionContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import '../style/connexion.css';
+import ConnexionNav from './connexionNav';
+import SignInForm from './signInForm';
+import SignUpForm from './sighUpForm';
+
+interface IProps {
+  text: string,
+  image: string,
+}
+
+interface IState {
+  text: string,
+  image: string,
+  form: any
```

Comment:
not sure about the any that should be a react component but this is not working. ideas ?